### ISSUE TITLE
Add GIS Day 2015 Pres to Site

### ISSUE THREAD

**Issue by powersa** [MaptimeSEA/maptimesea.github.io#12]
Link our presentation materials to the website.

**Comment by mapsam** [comment]
Agreed! Maybe we no longer call things "tutorials" and just start calling them "resources"?

**Comment by powersa** [comment]
I was thinking the same thing. Something like hand drawn maps ain't a tutorial. it's more of a resource how to type deal

**Comment by flott** [comment]
Added the slides and a link to the video in #34 . Better late than never!
